### PR TITLE
Dc 1721 new

### PR DESCRIPTION
**Pull request by lxol**

### CODE REVIEW

File: test/uk/gov/hmrc/ups/connectors/MockHttp.scala [hunk 1]
```diff
@@ -29,6 +32,12 @@ trait MockHttpGet extends MockitoSugar {
   val http = new HttpGet with WSHttp {
     override def doGet(url: String)(implicit hc: HeaderCarrier) =
       Future.successful(httpWrapper.getF(url))
+
+    override protected def appNameConfiguration: Configuration = ???
+
+    override protected def actorSystem: ActorSystem = ActorSystem("test-actor-system")
```
Comment (andy1138) [inline test/uk/gov/hmrc/ups/connectors/MockHttp.scala]: Do we need a new actor systems each time or could it be
private val anActorSystem = ActorSystem("test-actor-system")
..
override protected def actorSystem: ActorSystem = anActorSystem

File: test/uk/gov/hmrc/ups/connectors/MockHttp.scala [hunk 2]
```diff
@@ -51,6 +61,12 @@ trait MockHttpPost extends MockitoSugar {
     override def doFormPost(url: String, body: Map[String, Seq[String]])(implicit hc: HeaderCarrier): Future[HttpResponse] = ???
 
     override def doEmptyPost[A](url: String)(implicit hc: HeaderCarrier): Future[HttpResponse] = ???
+
+    override protected def appNameConfiguration: Configuration = ???
+
+    override protected def actorSystem: ActorSystem = ActorSystem("test-actor-system")
```
Comment (andy1138) [inline test/uk/gov/hmrc/ups/connectors/MockHttp.scala]: as above

File: test/uk/gov/hmrc/ups/connectors/MockHttp.scala (hunk 1 above)
Comment (lxol) [reply test/uk/gov/hmrc/ups/connectors/MockHttp.scala]: as far as I understand it wouldn't make any difference if I put it to the same trait

**Review by rwalpole — APPROVED** [review]

File: test/uk/gov/hmrc/ups/connectors/MockHttp.scala (hunk 1 above)
Comment (lxol) [reply test/uk/gov/hmrc/ups/connectors/MockHttp.scala]: fixed...

**Pull request merged**